src/main/config.cpp

```diff
@@ -449,6 +465,39 @@ idx_t DBConfig::ParseMemoryLimit(const string &arg) {
 	return NumericCast<idx_t>(multiplier * limit);
 }
 
+idx_t DBConfig::ParseMemoryLimitSlurm(const string &arg) {
+	if (arg.empty()) {
+		return 0;
+	}
+
+	string number_str = arg;
+	idx_t multiplier = 1000LL * 1000LL; // Default to MB if no unit specified
```

Comment:
Is the default multiplier the only difference here? If so, can we unify this code with the regular `DBConfig:: ParseMemoryLimit` by making the `default_multiplier` an `optional_idx`?

Reply:
The units are also different, I had it in one function initially but it looked very messy. I now removed DUCKDB_API since it is only an internal function